bundles/thunk_bundle.js

```diff
@@ -6207,7 +5802,7 @@ module.exports = (() => {
           }
           name = JSON.stringify("" + key);
           if (name.match(/^"([a-zA-Z_][a-zA-Z_0-9]*)"$/)) {
-            name = name.substr(1, name.length - 2);
+            name = name.slice(1, -1);
             name = ctx.stylize(name, "name");
           } else {
             name = name.replace(/'/g, "\\'").replace(/\\"/g, '"').replace(/(^"|"$)/g, "'");
```

Comment:
I can't explain all the differences here, but it's ~400 lines shorter and some of the differences I looked into seemed to be from using newer versions of dependencies